src/ToastNotification/ToastNotification.js

```diff
@@ -0,0 +1,65 @@
+import ClassNames from 'classnames'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * ToastNotification Component for Patternfly React
+ */
+const ToastNotification = ({
+  notificationClass,
+  children,
+  onDismiss,
+  type
+}) => {
+  const notificationClasses = ClassNames(
+    {
+      alert: true,
+      'toast-pf': true,
+      'alert-danger': type === 'danger' || type === 'error',
+      'alert-warning': type === 'warning',
+      'alert-success': type === 'success',
+      'alert-info': type === 'info',
+      'alert-dismissable': onDismiss
+    },
+    notificationClass
+  )
+  const iconClass = ClassNames({
+    pficon: true,
+    'pficon-error-circle-o': type === 'danger' || type === 'error',
+    'pficon-warning-triangle-o': type === 'warning',
+    'pficon-ok': type === 'success',
+    'pficon-info': type === 'info'
+  })
+
+  return (
+    <div className={notificationClasses}>
+      {onDismiss &&
+        <button
+          type="button"
+          className="close"
+          aria-hidden="true"
+          onClick={onDismiss}
+        >
+          <span className="pficon pficon-close" />
+        </button>}
+      <span className={iconClass} />
+      {children}
+    </div>
+  )
+}
+ToastNotification.propTypes = {
+  /** additional notification classes */
+  notificationClass: PropTypes.string,
+  /** callback when alert is dismissed  */
+  onDismiss: PropTypes.func,
+  /** the type of alert  */
+  type: PropTypes.oneOf(['danger', 'error', 'warning', 'success', 'info'])
```

Comment:
hmm not sure if this warrants a shared const, as it is singular to this component? (even though Alert is similar, in my mind they are different components sharing common css). It also conceivable these components change in the future. I'm fine with using a `const` if you want though ;)